crates/bevy_reflect/src/type_uuid.rs

```diff
@@ -16,4 +18,12 @@ where
     fn type_uuid(&self) -> Uuid {
         Self::TYPE_UUID
     }
+
+    fn display_type(&self) -> String {
```

Comment:
Maybe I'm just missing something obvious, but is there a reason you didn't use the `std::fmt::Display` trait here? Then you could call the standard `to_string()` method up above.